libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/SlackAdapter.cs

```diff
@@ -173,9 +181,9 @@ public override async Task<ResourceResponse[]> SendActivitiesAsync(ITurnContext
                             responses.Add(resourceResponse as ResourceResponse);
                         }
                     }
-                    catch (Exception ex)
+                    catch
```

Comment:
If we are not going to throw a new kind of exception, remove the `try/catch` block.